command/operator_debug.go

```diff
@@ -423,6 +429,20 @@ func (c *OperatorDebugCommand) Run(args []string) int {
 	}
 	c.index = uint64(eventIndex)
 
+	// Parse the pprof capture interval
+	pi, err := time.ParseDuration(pprofInterval)
+	if err != nil {
+		c.Ui.Error(fmt.Sprintf("Error parsing interval: %s: %s", pprofInterval, err.Error()))
+		return 1
+	}
+	c.pprofInterval = pi
+
+	// Validate interval
+	if pi.Seconds() > pd.Seconds() {
+		c.Ui.Error(fmt.Sprintf("Error parsing pprof interval: %s is greater than duration %s", interval, duration))
+		return 1
+	}
```

Comment:
I think we're reporting the wrong variables here, but also this is a validation error and not a parsing error. Comparing to the other validation errors above we've just left out that "Error parsing..." prefix for those:

```suggestion
	// Validate interval
	if pi.Seconds() > pd.Seconds() {
		c.Ui.Error(fmt.Sprintf("pprof-interval %s must be less than interval duration %s", pprofInterval, pprofDuration))
		return 1
	}
```

Also, this conditional is breaking all the `TestDebug_*` tests in this package (see this [`test-other`](https://app.circleci.com/pipelines/github/hashicorp/nomad/19460/workflows/b231cab3-8739-4556-ba33-b63bd6b4205b/jobs/197670) run). Those will need to be updated as well.